Tests/XcbeautifyLibTests/GitHubActionsLogTests.swift

```diff
@@ -0,0 +1,678 @@
+import XCTest
+@testable import XcbeautifyLib
+
+final class GitHubActionsLogTests: XCTestCase {
+    var parser: Parser!
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        parser = Parser(colored: false, outputFormat: .gitHubActions, additionalLines: { nil } )
+    }
+
+    private func logFormatted(_ string: String) -> String? {
+        return parser.parse(line: string)
+    }
+
+    func testAggregateTarget() {
+        let formatted = logFormatted("=== BUILD AGGREGATE TARGET Be Aggro OF PROJECT AggregateExample WITH CONFIGURATION Debug ===")
+        XCTAssertEqual(formatted, "Aggregate target Be Aggro of project AggregateExample with configuration Debug")
+        XCTAssertEqual(parser.outputType, .task)
+    }
+
+    func testAnalyze() {
+        let formatted = logFormatted("AnalyzeShallow /Users/admin/CocoaLumberjack/Classes/DDTTYLogger.m normal x86_64 (in target: CocoaLumberjack-Static)")
+        XCTAssertEqual(formatted, "[CocoaLumberjack-Static] Analyzing DDTTYLogger.m")
+        XCTAssertEqual(parser.outputType, .task)
+    }
+
+    func testAnalyzeTarget() {
+        let formatted = logFormatted("=== ANALYZE TARGET The Spacer OF PROJECT Pods WITH THE DEFAULT CONFIGURATION Debug ===")
+        XCTAssertEqual(formatted, "Analyze target The Spacer of project Pods with configuration Debug")
+        XCTAssertEqual(parser.outputType, .task)
+    }
+
+    func testBuildTarget() {
+        let formatted = logFormatted("=== BUILD TARGET The Spacer OF PROJECT Pods WITH THE DEFAULT CONFIGURATION Debug ===")
+        XCTAssertEqual(formatted, "Build target The Spacer of project Pods with configuration Debug")
+        XCTAssertEqual(parser.outputType, .task)
+    }
+
+    func testCheckDependenciesErrors() {
+    }
+
+    func testCheckDependencies() {
+    }
```

Comment:
Is the plan to add tests for those as well?

Reply:
I think this'll need to be a follow up PR because I don't know what the expected output is here. It wasn't defined in the pre-existing implementation (for TerminalRenderer).